### PR TITLE
[FEATURE] [MER-3239] need-prior-knowledge-check-container-name-added-to-list

### PR DESCRIPTION
**Pull request by tomasferok**
this pr [MER-3239](https://eliterate.atlassian.net/browse/MER-3239) add a new label to Group activities
Screenshot 2024-05-08 at 09 01 48

[MER-3239]: https://eliterate.atlassian.net/browse/MER-3239?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: assets/src/data/content/resource.ts [hunk 1]
```diff
@@ -180,6 +180,7 @@ export const PurposeTypes: Purpose[] = [
   { value: 'quiz', label: 'Quiz' },
   { value: 'simulation', label: 'Simulation' },
   { value: 'walkthrough', label: 'Walkthrough' },
+  { value: 'priorknowledgecheck', label: 'Prior Knowledge Check' },
```
Comment (nicocirio) [inline assets/src/data/content/resource.ts:183]: NIT: WDYT about adding this option after the "Many students wonder"'s option to keep them sorted alphabetically?

**Review by nicocirio — COMMENTED** [review]
Nice work Tomi!

File: assets/src/data/content/resource.ts (hunk 1 above)
Comment (tomasferok) [reply assets/src/data/content/resource.ts]: I just pushed your changes thnks

**Review by nicocirio — APPROVED** [review]
🚀

**Review by darrensiegel — APPROVED** [review]

**Pull request merged**